recipe/build.sh

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env bash
+# Get an updated config.sub and config.guess
+cp $BUILD_PREFIX/share/gnuconfig/config.* ./nspr/build/autoconf
```

Comment:
Not sure if this is the reason of the failure, but the blog has the following line according to https://conda-forge.org/blog/posts/2020-10-29-macos-arm64/#how-to-add-a-osx-arm64-build-to-a-feedstock
`cp $BUILD_PREFIX/share/libtool/build-aux/config.* .`